frontend/src/metabase/components/MoveQuestionsIntoDashboardsModal/ConfirmMoveDashboardQuestionCandidatesModal.tsx

```diff
@@ -0,0 +1,164 @@
+      return [];
+    }
+
+    return candidates.map(candidate => ({
+      id: candidate.id,
+      questionName: candidate.name,
+      dashboardName: candidate.sole_dashboard_info.name,
+    }));
+  }, [isLoading, fetchError, candidates]);
+
+  const defaultErrMsg = t`Something went wrong`;
+
+  const ctaDisabled = !!(
+    isMutating ||
+    isLoading ||
+    fetchError ||
+    rows.length === 0
+  );
+
+  return (
+    <Modal.Root
+      opened
+      onClose={onCancel}
+      data-testid="move-questions-into-dashboard-modal"
+      size="64rem"
+    >
+      <Modal.Overlay />
+      <Modal.Content className={S.modal}>
+        <Modal.Header
+          px="2.5rem"
+          pt="2rem"
+          pb="1.5rem"
+          className={S.modalHeader}
+        >
+          <Modal.Title fz="20px">
+            {fetchError || isLoading || rows.length === 0
+              ? t`Move these questions into their dashboards?`
+              : ngettext(
+                  msgid`Move this question into its dashboard?`,
+                  `Move these ${rows.length} questions into their dashboards?`,
+                  rows.length,
+                )}
+          </Modal.Title>
+          <Modal.CloseButton data-testid="move-questions-into-dashboard-modal-close-btn" />
+        </Modal.Header>
+        <Modal.Body p="0" className={S.modalBody}>
+          <div className={S.tableHeader}>
+            <div className={S.tableRow}>
+              <div className={S.column}>
+                <Flex gap="sm" align="center">
+                  <Icon name="folder" c="brand" />
+                  {t`Saved Question`}
+                </Flex>
+              </div>
+              <div className={S.column}>
+                <Flex gap="sm" align="center">
+                  <Icon name="dashboard" c="brand" />
+                  {t`Dashboard it'll be moved to`}
+                </Flex>
+              </div>
+            </div>
+          </div>
+          <div className={S.tbody}>
+            {match({ isLoading, fetchError, rows })
+              .with({ isLoading: true }, () => (
+                <Flex justify="center" py="18.25rem">
+                  <Loader size="xl" data-testid="loader" />
+                </Flex>
+              ))
+              .with({ fetchError: P.not(P.nullish) }, ({ fetchError }) => {
+                return (
+                  <Flex justify="center" py="19rem">
+                    <Text color="error" size="1.25rem" px="md">
+                      {fetchError instanceof Error
+                        ? (fetchError?.message ?? defaultErrMsg)
+                        : defaultErrMsg}
+                    </Text>
+                  </Flex>
+                );
+              })
+              .with({ rows: [] }, () => (
+                <Flex justify="center" py="19rem">
+                  <Text size="1.25rem" px="md" color="text-light">
+                    {t`There aren't any questions to clean up! Looks like everything is in its place.`}
```

Comment:
```suggestion
                    {t`There aren't any questions to move into dashboards. Looks like everything is in its place.`}
```
Just a tad more specific about what we mean by "clean up" to distinguish this workflow from the one that moves items to the trash.

I don't know about the exclamation point, so I dropped it.